Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -552,7 +551,7 @@ protected object CreateInstanceSafe(Tuple<Type, string> typeInfo)
         protected void RegisterSingleInstance<T>(T obj, bool manageLifetime = true)
             where T : class
         {
-            Container.RegisterSingleton(obj);
+            Container.RegisterInstance<T>(obj);
```

Comment:
Care to explain what this code does? I wonder if this potential change in behaviour (from registering something that is singleton to registering an instance of something) could break stuff.

Reply:
The old function was obsolete.